lib/synapse/haproxy.rb

```diff
@@ -1,12 +1,18 @@
 require 'fileutils'
 require 'json'
 require 'socket'
+require 'digest/sha1'
 
 module Synapse
   class Haproxy
     include Logging
     attr_reader :opts, :name
 
+    @@cookie_value_methods = [
```

Comment:
Let's not actually validate this. Similar to `backend_order` option let's just have a case statement with an else that provides the default behavior.